### PR TITLE
SignIn UI completed

### PR DESCRIPTION
**Pull request by Pavithrank02**

### CODE REVIEW

File: src/SignIn.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import TextField from '@mui/material/TextField';
+import GoogleIcon from '@mui/icons-material/Google';
+import TwitterIcon from '@mui/icons-material/Twitter';
+import "./Styles.css";
+import AccountCircleIcon from '@mui/icons-material/AccountCircle';import Home from "./Home";
+
+
+function SignIn() {
+  return (
+    <div className="signin">
```
Comment (gowrishankkar) [inline src/SignIn.jsx]: Use Grid

File: src/Styles.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  flex-direction: column;
+  background-color: rgba(255, 255, 255, 0.822);
+  align-items: center;
+  justify-content: center;
+  color: rgb(0, 0, 0);
+  border-radius: 10px;
+  border: 2px solid blue;
+  height: 65vh;
+}
+form{
```
Comment (gowrishankkar) [inline src/Styles.css]: Dont use form tag

File: src/Styles.css [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+.signin > .log{
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  width: 20vw;
+  margin-bottom: 20px;
+  cursor: pointer;
+}
+
+.home > a{
```
Comment (gowrishankkar) [inline src/Styles.css:66]: use typography

**Review by gowrishankkar — CHANGES_REQUESTED** [review]

File: src/SignIn.jsx (hunk 1 above)
Comment (Pavithrank02) [reply src/SignIn.jsx]: completed

File: src/Styles.css (hunk 2 above)
Comment (Pavithrank02) [reply src/Styles.css]: completed

File: src/Styles.css (hunk 3 above)
Comment (Pavithrank02) [reply src/Styles.css:66]: completed

Comment (gowrishankkar) [reply src/Styles.css:66]: If you've used typography, then remove this css

**Review by gowrishankkar — APPROVED** [review]

**Pull request merged**